### ISSUE TITLE
Updater updating every single app???

### ISSUE THREAD

**Issue by mobilegmYT** [Botspot/pi-apps#663]
I just ran the updater... on the update list, it seemingly has every single app listed. Below is terminal output

```
pi@TwisterOS-RPN:~/Downloads $ ~/pi-apps/updater
Checking for updates now...
Cloning into 'pi-apps'...
remote: Enumerating objects: 887, done.
remote: Counting objects: 100% (887/887), done.
remote: Compressing objects: 100% (709/709), done.
remote: Total 887 (delta 116), reused 678 (delta 72), pack-reused 0
Receiving objects: 100% (887/887), 1.33 MiB | 2.88 MiB/s, done.
Resolving deltas: 100% (116/116), done.
All Is Well exists in both locations, but online version is newer. Adding to updatable list.
AndroidBuddy exists in both locations, but online version is newer. Adding to updatable list.
Angry IP scanner exists in both locations, but online version is newer. Adding to updatable list.
AnyDesk exists in both locations, but online version is newer. Adding to updatable list.
Arduino exists in both locations, but online version is newer. Adding to updatable list.
AstroMenace exists in both locations, but online version is newer. Adding to updatable list.
ATlauncherPI only exists locally. Will not add to updatable list.
BalenaEtcher exists in both locations, but online version is newer. Adding to updatable list.
BlockPi exists in both locations, but online version is newer. Adding to updatable list.
Bongo Cam exists in both locations, but online version is newer. Adding to updatable list.
Box86 exists in both locations, but online version is newer. Adding to updatable list.
Boxy SVG exists in both locations, but online version is newer. Adding to updatable list.
Browsh exists in both locations, but online version is newer. Adding to updatable list.
Chiaki exists in both locations, but online version is newer. Adding to updatable list.
Chromium Widevine exists in both locations, but online version is newer. Adding to updatable list.
Clam Antivirus exists in both locations, but online version is newer. Adding to updatable list.
Color Emoji font exists in both locations, but online version is newer. Adding to updatable list.
CommanderPi exists in both locations, but online version is newer. Adding to updatable list.
Conky exists in both locations, but online version is newer. Adding to updatable list.
Cool Retro Term exists in both locations, but online version is newer. Adding to updatable list.
Cura exists in both locations, but online version is newer. Adding to updatable list.
Descent 1 exists in both locations, but online version is newer. Adding to updatable list.
Descent 2 exists in both locations, but online version is newer. Adding to updatable list.
Discord exists in both locations, but online version is newer. Adding to updatable list.
Doom 3 exists in both locations, but online version is newer. Adding to updatable list.
Downgrade Chromium exists in both locations, but online version is newer. Adding to updatable list.
Drawing exists in both locations, but online version is newer. Adding to updatable list.
Eagle CAD exists in both locations, but online version is newer. Adding to updatable list.
eDEX-UI exists in both locations, but online version is newer. Adding to updatable list.
Email Checker exists in both locations, but online version is newer. Adding to updatable list.
Flameshot exists in both locations, but online version is newer. Adding to updatable list.
FreeCAD exists in both locations, but online version is newer. Adding to updatable list.
FreeTube exists in both locations, but online version is newer. Adding to updatable list.
Geany Dark Mode exists in both locations, but online version is newer. Adding to updatable list.
Github-CLI exists in both locations, but online version is newer. Adding to updatable list.
Heroes 2 exists in both locations, but online version is newer. Adding to updatable list.
Intellij IDEA exists in both locations, but online version is newer. Adding to updatable list.
jGRASP IDE exists in both locations, but online version is newer. Adding to updatable list.
Lego Digital Designer exists in both locations, but online version is newer. Adding to updatable list.
Libreoffice MS theme exists in both locations, but online version is newer. Adding to updatable list.
LibrePCB exists in both locations, but online version is newer. Adding to updatable list.
Lightpad exists in both locations, but online version is newer. Adding to updatable list.
Lokinet exists in both locations, but online version is newer. Adding to updatable list.
Lutris only exists locally. Will not add to updatable list.
Mac OS Theme exists in both locations, but online version is newer. Adding to updatable list.
Minecraft Bedrock exists in both locations, but online version is newer. Adding to updatable list.
Minecraft Java exists in both locations, but online version is newer. Adding to updatable list.
Minecraft Pi (Modded) exists in both locations, but online version is newer. Adding to updatable list.
Mission Planner exists in both locations, but online version is newer. Adding to updatable list.
Node.js exists in both locations, but online version is newer. Adding to updatable list.
Notepad ++ exists in both locations, but online version is newer. Adding to updatable list.
OBS Studio exists in both locations, but online version is newer. Adding to updatable list.
OpenSCAD exists in both locations, but online version is newer. Adding to updatable list.
Pi-Apps Terminal Plugin exists in both locations, but online version is newer. Adding to updatable list.
PiGro exists in both locations, but online version is newer. Adding to updatable list.
piKiss exists in both locations, but online version is newer. Adding to updatable list.
Pi Power Tools exists in both locations, but online version is newer. Adding to updatable list.
PiSafe exists in both locations, but online version is newer. Adding to updatable list.
Powerline-Shell exists in both locations, but online version is newer. Adding to updatable list.
PPSSPP (PSP emulator) exists in both locations, but online version is newer. Adding to updatable list.
Processing IDE exists in both locations, but online version is newer. Adding to updatable list.
PrusaSlicer exists in both locations, but online version is newer. Adding to updatable list.
Puffin Browser Demo exists in both locations, but online version is newer. Adding to updatable list.
Pycharm CE exists in both locations, but online version is newer. Adding to updatable list.
QEMU exists in both locations, but online version is newer. Adding to updatable list.
Raspi2png exists in both locations, but online version is newer. Adding to updatable list.
Remarkable exists in both locations, but online version is newer. Adding to updatable list.
Scratch 2 exists in both locations, but online version is newer. Adding to updatable list.
Scrcpy exists in both locations, but online version is newer. Adding to updatable list.
Snapdrop exists in both locations, but online version is newer. Adding to updatable list.
Snap Store exists in both locations, but online version is newer. Adding to updatable list.
Sonic Pi 3.2 exists in both locations, but online version is newer. Adding to updatable list.
SpeedTest-CLI exists in both locations, but online version is newer. Adding to updatable list.
StackEdit exists in both locations, but online version is newer. Adding to updatable list.
Steam exists in both locations, but online version is newer. Adding to updatable list.
Stunt Rally exists in both locations, but online version is newer. Adding to updatable list.
Sublime Text exists in both locations, but online version is newer. Adding to updatable list.
TBOPlayer exists in both locations, but online version is newer. Adding to updatable list.
TeamViewer Host exists in both locations, but online version is newer. Adding to updatable list.
template exists in both locations, but online version is newer. Adding to updatable list.
Timeshift exists in both locations, but online version is newer. Adding to updatable list.
tldr exists in both locations, but online version is newer. Adding to updatable list.
Tor Browser exists in both locations, but online version is newer. Adding to updatable list.
Turbo Scratch exists in both locations, but online version is newer. Adding to updatable list.
Ulauncher exists in both locations, but online version is newer. Adding to updatable list.
Update Buddy exists in both locations, but online version is newer. Adding to updatable list.
USBImager exists in both locations, but online version is newer. Adding to updatable list.
VeraCrypt exists in both locations, but online version is newer. Adding to updatable list.
Visual Studio Code exists in both locations, but online version is newer. Adding to updatable list.
Vivaldi exists in both locations, but online version is newer. Adding to updatable list.
VSCodium exists in both locations, but online version is newer. Adding to updatable list.
WACUP (new WinAmp) exists in both locations, but online version is newer. Adding to updatable list.
Web Apps exists in both locations, but online version is newer. Adding to updatable list.
Wechat exists in both locations, but online version is newer. Adding to updatable list.
Whatsapp exists in both locations, but online version is newer. Adding to updatable list.
Windows 10 Theme exists in both locations, but online version is newer. Adding to updatable list.
Windows Screensavers exists in both locations, but online version is newer. Adding to updatable list.
Wine (x86) exists in both locations, but online version is newer. Adding to updatable list.
WPS Office exists in both locations, but online version is newer. Adding to updatable list.
YouTubuddy exists in both locations, but online version is newer. Adding to updatable list.
Zoom exists in both locations, but online version is newer. Adding to updatable list.
updatable: All Is Well|AndroidBuddy|Angry IP scanner|AnyDesk|Arduino|AstroMenace|BalenaEtcher|BlockPi|Bongo Cam|Box86|Boxy SVG|Browsh|Chiaki|Chromium Widevine|Clam Antivirus|Color Emoji font|CommanderPi|Conky|Cool Retro Term|Cura|Descent 1|Descent 2|Discord|Doom 3|Downgrade Chromium|Drawing|Eagle CAD|eDEX-UI|Email Checker|Flameshot|FreeCAD|FreeTube|Geany Dark Mode|Github-CLI|Heroes 2|Intellij IDEA|jGRASP IDE|Lego Digital Designer|Libreoffice MS theme|LibrePCB|Lightpad|Lokinet|Mac OS Theme|Minecraft Bedrock|Minecraft Java|Minecraft Pi (Modded)|Mission Planner|Node.js|Notepad ++|OBS Studio|OpenSCAD|Pi-Apps Terminal Plugin|PiGro|piKiss|Pi Power Tools|PiSafe|Powerline-Shell|PPSSPP (PSP emulator)|Processing IDE|PrusaSlicer|Puffin Browser Demo|Pycharm CE|QEMU|Raspi2png|Remarkable|Scratch 2|Scrcpy|Snapdrop|Snap Store|Sonic Pi 3.2|SpeedTest-CLI|StackEdit|Steam|Stunt Rally|Sublime Text|TBOPlayer|TeamViewer Host|template|Timeshift|tldr|Tor Browser|Turbo Scratch|Ulauncher|Update Buddy|USBImager|VeraCrypt|Visual Studio Code|Vivaldi|VSCodium|WACUP (new WinAmp)|Web Apps|Wechat|Whatsapp|Windows 10 Theme|Windows Screensavers|Wine (x86)|WPS Office|YouTubuddy|Zoom
Excluding 'apps/Wine (x86)' from the mergedlist.
Excluding 'apps/Steam' from the mergedlist.
api is identical to the online version. Nothing to do!
CHANGELOG.md is identical to the online version. Nothing to do!
COPYING is identical to the online version. Nothing to do!
createapp is identical to the online version. Nothing to do!
etc/bitlylink is identical to the online version. Nothing to do!
etc/categoryedit is identical to the online version. Nothing to do!
etc/git_url is identical to the online version. Nothing to do!
etc/preload-daemon is identical to the online version. Nothing to do!
etc/setting-params/App List Style is identical to the online version. Nothing to do!
etc/setting-params/Check for updates is identical to the online version. Nothing to do!
etc/setting-params/Enable analytics is identical to the online version. Nothing to do!
etc/setting-params/Preferred text editor is identical to the online version. Nothing to do!
etc/setting-params/Reinstall after update only exists locally. Will not add to updatable list.
etc/setting-params/Show Edit button is identical to the online version. Nothing to do!
etc/setting-params/Shuffle App list is identical to the online version. Nothing to do!
etc/terminal-run is identical to the online version. Nothing to do!
.github/FUNDING.yml is identical to the online version. Nothing to do!
.github/ISSUE_TEMPLATE/app-request.md is identical to the online version. Nothing to do!
.github/ISSUE_TEMPLATE/bug_report.md is identical to the online version. Nothing to do!
.github/ISSUE_TEMPLATE/suggestion.md is identical to the online version. Nothing to do!
gui is identical to the online version. Nothing to do!
icons/back-64.png is identical to the online version. Nothing to do!
icons/background-3d-dark.png is identical to the online version. Nothing to do!
icons/background-3d-dark.svg only exists locally. Will not add to updatable list.
icons/background-3d.png is identical to the online version. Nothing to do!
icons/background-3d.svg only exists locally. Will not add to updatable list.
icons/back.png is identical to the online version. Nothing to do!
icons/back.svg only exists locally. Will not add to updatable list.
icons/backup.png is identical to the online version. Nothing to do!
icons/badge.png is identical to the online version. Nothing to do!
icons/badge.svg only exists locally. Will not add to updatable list.
icons/categories/All Apps-64.png is identical to the online version. Nothing to do!
icons/categories/All Apps.png is identical to the online version. Nothing to do!
icons/categories/default-64.png is identical to the online version. Nothing to do!
icons/categories/default.png is identical to the online version. Nothing to do!
icons/categories/Installed-64.png is identical to the online version. Nothing to do!
icons/categories/Installed.png is identical to the online version. Nothing to do!
icons/categories.png is identical to the online version. Nothing to do!
icons/check.png is identical to the online version. Nothing to do!
icons/corrupted.png is identical to the online version. Nothing to do!
icons/create.png is identical to the online version. Nothing to do!
icons/credits.png only exists locally. Will not add to updatable list.
icons/disabled.png is identical to the online version. Nothing to do!
icons/download.png is identical to the online version. Nothing to do!
icons/down.png is identical to the online version. Nothing to do!
icons/edit.png is identical to the online version. Nothing to do!
icons/exit.png is identical to the online version. Nothing to do!
icons/folder-64.png only exists locally. Will not add to updatable list.
icons/folder-installed-64.png only exists locally. Will not add to updatable list.
icons/folder-installed.png only exists locally. Will not add to updatable list.
icons/folder.png only exists locally. Will not add to updatable list.
icons/forward.png is identical to the online version. Nothing to do!
icons/image.png is identical to the online version. Nothing to do!
icons/info.png is identical to the online version. Nothing to do!
icons/in-progress.png is identical to the online version. Nothing to do!
icons/in_progress.svg only exists locally. Will not add to updatable list.
icons/installed.png is identical to the online version. Nothing to do!
icons/install.png is identical to the online version. Nothing to do!
icons/logo-128.png only exists locally. Will not add to updatable list.
icons/logo-128-trans.png is identical to the online version. Nothing to do!
icons/logo-128-trans.svg only exists locally. Will not add to updatable list.
icons/logo-3d-dark.png is identical to the online version. Nothing to do!
icons/logo-3d-dark.svg only exists locally. Will not add to updatable list.
icons/logo-3d.png is identical to the online version. Nothing to do!
icons/logo-3d.svg only exists locally. Will not add to updatable list.
icons/logo-64.png is identical to the online version. Nothing to do!
icons/logo.png is identical to the online version. Nothing to do!
icons/logo.svg only exists locally. Will not add to updatable list.
icons/logo-trans.png is identical to the online version. Nothing to do!
icons/logo-trans.svg only exists locally. Will not add to updatable list.
icons/none-24.png is identical to the online version. Nothing to do!
icons/none-64.png is identical to the online version. Nothing to do!
icons/none.png is identical to the online version. Nothing to do!
icons/paypal.png is identical to the online version. Nothing to do!
icons/proglogo.png is identical to the online version. Nothing to do!
icons/proglogo.svg only exists locally. Will not add to updatable list.
icons/save.png is identical to the online version. Nothing to do!
icons/screenshots/app list.png is identical to the online version. Nothing to do!
icons/screenshots/category editor.png is identical to the online version. Nothing to do!
icons/screenshots/create app.png is identical to the online version. Nothing to do!
icons/screenshots/details window.png is identical to the online version. Nothing to do!
icons/screenshots/main window.png is identical to the online version. Nothing to do!
icons/screenshots/settings.png is identical to the online version. Nothing to do!
icons/screenshots/updates available.png is identical to the online version. Nothing to do!
icons/shellscript.png is identical to the online version. Nothing to do!
icons/trash.png is identical to the online version. Nothing to do!
icons/txt.png is identical to the online version. Nothing to do!
icons/uninstalled.png is identical to the online version. Nothing to do!
icons/uninstall.png is identical to the online version. Nothing to do!
icons/up.png is identical to the online version. Nothing to do!
icons/vector/background-3d-dark.svg is identical to the online version. Nothing to do!
icons/vector/background-3d.svg is identical to the online version. Nothing to do!
icons/vector/back.svg is identical to the online version. Nothing to do!
icons/vector/badge.svg is identical to the online version. Nothing to do!
icons/vector/in_progress.svg is identical to the online version. Nothing to do!
icons/vector/logo-128-trans.svg is identical to the online version. Nothing to do!
icons/vector/logo-3d-dark.svg is identical to the online version. Nothing to do!
icons/vector/logo-3d.svg is identical to the online version. Nothing to do!
icons/vector/logo.svg is identical to the online version. Nothing to do!
icons/vector/logo-trans.svg is identical to the online version. Nothing to do!
icons/vector/proglogo.svg is identical to the online version. Nothing to do!
install is identical to the online version. Nothing to do!
manage is identical to the online version. Nothing to do!
pi-apps-terminal-bash-edition.sh only exists locally. Will not add to updatable list.
pkg-install is identical to the online version. Nothing to do!
preload is identical to the online version. Nothing to do!
purge-installed is identical to the online version. Nothing to do!
README.md is identical to the online version. Nothing to do!
settings is identical to the online version. Nothing to do!
uninstall is identical to the online version. Nothing to do!
updater exists in both locations, but online version is newer. Adding to updatable list.
```

**Comment by Botspot** [comment]
Thanks for catching that. Hopefully this was the only mistake made in today's many changes.
